knowage/src/main/java/it/eng/spago/dispatching/httpchannel/AdapterHTTP.java

```diff
@@ -579,19 +577,11 @@ private void setHttpRequestData(HttpServletRequest request, RequestContainer req
 
 	// RequestContainer requestContainer)
 
-	/*
-	 * (non-Javadoc)
-	 *
-	 * @see javax.servlet.GenericServlet#init()
-	 */
-	@Override
-	public void init() throws ServletException {
-		super.init();
-
+	private boolean getSerializedSession() {
```

Comment:
It's ok but pay attention: the init() method is called only once; that means that in the past the value was read only one time at the boot of the application from a source which let you know the value of a changeable configuration. The code was in strong contradiction as you can say: i think nobody knows what is the right way to manage that value. Anyway this is the less worse way to manage this, good job.